### PR TITLE
ign -> gz Provisional Finale: Source Migration : gz-transport

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/767

### CODE REVIEW

File: include/gz/transport/CIface.h [hunk 1]
```diff
@@ -30,29 +30,30 @@ extern "C" {
   } SubscribeOpts;
 
   /// \brief A transport node.
-  typedef struct IgnTransportNode IgnTransportNode;
+  typedef struct GzTransportNode GzTransportNode;
+  using IgnTransportNode GZ_DEPRECATED(12) = GzTransportNode;  // TODO(CH3): Deprecated. Remove!
```
Comment (chapulina) [inline include/gz/transport/CIface.h:34]: There's something here that Windows doesn't like

```
C:\Jenkins\workspace\ign_transport-pr-win\ws\ign-transport\include\gz/transport/CIface.h(34,26): error C2062: type 'int' unexpected (compiling source file C:\Jenkins\workspace\ign_transport-pr-win\ws\ign-transport\src\CIface.cc) 
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: include/gz/transport/CIface.h (hunk 1 above)
Comment (methylDragon) [reply include/gz/transport/CIface.h:34]: I'm pretty sure it's the `GZ_DEPRECATED` macro call...

I couldn't get it to compile putting it anywhere else, but I guess windows doesn't like it even there.
Are you alright with removing it?

Comment (methylDragon) [reply include/gz/transport/CIface.h]: https://github.com/gazebosim/gz-transport/pull/329/commits/350ecf04f8c2aa7e38256cb7fe6902c81d506d44 fixed like the rest

**Review by chapulina — APPROVED** [review]

**Comment by chapulina** [comment]
The failures aren't caused by this PR and are being addressed in https://github.com/gazebosim/gz-transport/pull/335.

**Pull request merged**